csaf-validation/src/main/kotlin/io/github/csaf/sbom/validation/Requirement.kt

```diff
@@ -87,11 +101,11 @@ infix fun Requirement.or(other: Requirement): Requirement {
     return Or(this, other)
 }
 
-internal class Or(var lhs: Requirement, var rhs: Requirement) : Requirement {
+internal class Or(private val lhs: Requirement, private val rhs: Requirement) : Requirement {
```

Comment:
Documentation. Somehow redundant to `OneOf` but that's closer to the standard I guess. Maybe share the implementation?

Reply:
I got rid of `Or` and replaced it with OneOf, I am keeping the public function `or` and rerouting it to oneof